app/components/Button/Button.css

```diff
@@ -1,52 +1,68 @@
 @import '~app/styles/variables.css';
 
 .button {
-  box-shadow: 1px 1px 3px 1px var(--color-gray-5);
-  font-weight: 600;
-  background: linear-gradient(
-    -180deg,
-    var(--color-red-3) 0%,
-    var(--color-red-2) 90%
-  );
-  padding: 10px 20px;
-  color: var(--color-light-gray-4);
-  outline: none;
-  border-radius: 4px;
+  display: inline-flex;
   align-items: center;
   justify-content: center;
-  display: inline-flex;
+  box-sizing: border-box;
+  height: 2.3em;
+  width: fit-content;
+  padding: 5px 15px;
+  box-shadow: 0 1px 0 rgba(27, 31, 36, 10%),
+    inset 0 1px 0 rgba(255, 255, 255, 3%);
+  font-weight: 600;
+  background-color: var(--color-light-gray-4);
+  color: var(--color-dark-gray-5);
+  outline: none;
+  border-radius: 6px;
+  border: 1px solid #30404d77;
+  transition: 125ms cubic-bezier(0.3, 0, 0.5, 1) ease-in-out;
+  /* stylelint-disable value-list-comma-newline-after */
+  transition-property: color, background-color, border-color, box-shadow,
+    opacity;
 
   &:active {
     transform: translateY(1px);
   }
 
+  &:focus {
+    box-shadow: var(--color-dark-mono-gray-4);
+  }
+
   &:disabled {
-    color: var(--color-dark-mono-gray-4);
-    background: var(--color-mono-gray-2);
     box-shadow: initial;
+    cursor: not-allowed;
+    opacity: 0.5;
   }
 
   + .button {
     margin-left: 16px;
   }
 
-  &:hover:not(:disabled) {
-    box-shadow: 0 1px 2px rgba(0, 0, 0, 10%);
-    opacity: 0.9;
-  }
-
   > i {
-    margin-right: 4px;
+    margin-right: 7px;
   }
 
   > a {
     color: var(--color-dark-gray-3);
   }
+
+  &:hover:not(:disabled) {
+    background: var(--color-light-gray-3);
+    border-color: #394b59aa;
```

Comment:
Is this supposed to be the same for dark and light mode?
And is there no variable that can be used for this?

Reply:
I would love to use a variable and I tried, but since our color variables are on hex format (and not rgb), it's not possible to apply an alpha value on the variable itself. I'm considering changing all the variables to rgb 🤔

Reply:
aaah I see now. Yeah I had the same issue with the dark mode update. So if you want to change some variables to rbg, I would say go ahead.

Reply:
Changed them now 👍🏼